audio/out/ao_pipewire.c

```diff
@@ -416,15 +431,18 @@ static int pipewire_init_boilerplate(struct ao *ao)
         goto error;
     }
 
-    ret = 0;
-
-out:
     pw_thread_loop_unlock(p->loop);
-    return ret;
+
+    if (!session_has_sinks(ao)) {
+        MP_INFO(ao, "PipeWire does not have any audio sinks, skipping\n");
```

Comment:
Use `MP_VERBOSE`. We shouldn't print messages from things we skip during probing.